external_config/slack.md

```diff
@@ -156,6 +156,14 @@ This should be added as a **global** shortcut.
 - **Short Description:** Show voters needing attention
 - **Callback ID:** show_needs_attention
 
+#### Open / Close Channels
+
+This should be added as a **global** shortcut.
+
+- **Name:** Open / Close Channels
```

Comment:
One minor suggestion: What are your thoughts on naming this `Manage Entry Points` (since you've included the weight functionality and an admin could change weights without completely opening/closing)?